.cspell.json

```diff
@@ -259,6 +263,7 @@
     "spworkflowprocesslist",
     "spworkspacepagelist",
     "stenciljs",
+    "TDIDF",
```

Comment:
nudge @coveo/api-writers : TDIDF should probably be replaced by TF-IDF in all github.com/coveo
https://github.com/search?q=org%3Acoveo%20TDIDF&type=code
(at least TDIDF with google is autocorrected to TF-IDF)

I chose to keep the potential typo to stay consistent with the REST APIs, as I think that's the lesser evil

Reply:
Ooops. Thanks for highlighting this. I created a JIRA to keep track of the issue: https://coveord.atlassian.net/browse/DOC-13421